idd/Energy+.idd.in

```diff
@@ -9261,10 +9261,13 @@ Space,
        \min-fields 3
        \extensible:1
   A1,  \field Name
+       \note Name of the Space.
+       \note This name must be unique across Zone, Space, ZoneList, and SpaceList names.
        \required-field
        \type alpha
        \reference SpaceNames
        \reference SpaceAndSpaceListNames
+       \reference ZoneAndZoneListAndSpaceAndSpaceListNames
```

Comment:
OpenStudio assumes this combined reference list name requires uniqueness across the objects that reference it.